src/style.css

```diff
@@ -0,0 +1,23 @@
+body {
+  margin: 20px;
+  background-color: black;
+}
+
+div {
+  border: 1px solid #e75551;
+  padding: 10px;
+}
+
+.blue-box {
```

Comment:
ok, in this case, because we do not have context, but classes with color in the name is not the best approach. What if tomorrow designer says: "the first block should be purple". You will need to change index.html to change class name, and change styles. A better name will be more agnostic one, `paragraph-wrapper` for example.